README.md

```diff
@@ -63,1498 +104,468 @@ Delegating a capability is like the statement `message`. Task is akin to `messag
+    Receipt |o--|{ Task: enqueues
+```
+
+## Anatomy
+[Anatomy]: #anatomy
+
+| Concept      | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| [Command]    | Function application; a description of work to be performed                 |
+| [Task]       | Contextual information for a [Command], such as resource limits             |
+| [Invocation] | A request to perform some [Task] based on [delegated][Delegation] authority |
+
+A request for some work to be done (or to "exercise your authority") is an Invocation.
+
+``` mermaid
+flowchart TD
+    subgraph Invocation
+        SignatureBytes["Signature (raw bytes)"]
+      
+        subgraph SigPayload ["Signature Payload"]
+            VarsigHeader["Varsig Header"]
+
+            subgraph InvocationPayload ["Invocation Payload"]
+                iss
+                sub
+                do
+                args
+                prf
+                cause["cause (optional)"]
+                etc["..."]
+            end
+        end
+    end
+    
+    cause -.->|CID| Receipt
+```
 
-type SemVer string
+As [noted in the introduction][lazy-vs-eager], there is a difference between a reference to a function and calling that function. The [Invocation] is a request to the [Executor] to perform the enclosed [Task]. [Invocation Payload]s are not executable until they have been signed and [Delegation] proofs validated.
 
-type Receipt struct {
-  # Invocation this is a receipt for
-  ran     &Invocation
+Note that the Invocation MUST include the Signature envelope. An [Invocation Payload] on its own MUST NOT be considered a valid Invocation.
 
-  # Output of the invocation
-  out     Result
+# [UCAN Envelope] Configuration
+[UCAN Envelope Configuration]: #ucan-envelope-configuration
 
-  # Effects to be performed
-  fx      Effects
+## Type Tag
+[Type Tag]: #type-tag
  
-  # All the other metadata
-  meta    {String : Any}
-
-  # Principal that issued this receipt. If omitted issuer is
-  # inferred from the invocation task audience.
-  iss     optional Principal
-
-  # When issuer is different from executor this MUST hold a UCAN
-  # delegation chain from executor to the issuer. This should be 
-  # omitted when the executor is the issuer.
-  prf     [&UCAN]
-
-  # Signature from the "iss".
-  s       Varsig
-}
-
-type Result union {
-  | any    "ok"    # Success
-  | any    "error" # Error
-} representation kinded
-
-# Represents a request to invoke enclosed set of tasks concurrently
-type Effects struct {
-  # Primary set of tasks to be invoked
-  fork      [&Task]
-  
-  # Continuation for straight-line programs
-  join       optional &Task
-}
-
-# Way to reference result of the Task
-type Await union {
-  | &Task    "await/*"
-  | &Task    "await/ok"
-  | &Task    "await/error"
-} representation keyed
-```
+The UCAN envelope's payload tag MUST be `ucan/inv@1.0.0-rc.1`.
```

Comment:
```suggestion
The UCAN envelope's [payload tag](https://github.com/ucan-wg/spec/blob/main/README.md#envelope) MUST be `ucan/inv@1.0.0-rc.1`.
```
Note that this^ URL won't be valid until [spec#173](https://github.com/ucan-wg/spec/pull/173) is merged, which includes the envelope version string.  (When do be bikeshed that field name, btw, because even writing this comment required a LOT of thought).

Reply:
Thanks! Merged manually to keep external links at the bottom of the doc